ros_github_scripts/ci_for_pr.py

```diff
@@ -378,6 +382,9 @@ def main():
             extra_build_args += f' --packages-above-and-dependencies {packages_changed}'
             extra_test_args += f' --packages-above {packages_changed}'
 
+    if parsed.cmake_args:
+        extra_build_args += f' --cmake-args -D{parsed.cmake_args}'
```

Comment:
This should probably be:

```suggestion
        extra_build_args += f' --cmake-args {parsed.cmake_args}'
```

otherwise passing `--cmake-args "-DSOME_VAR=ON -DOTHER_VAR=ON"` to this script won't work. You'd need to pass `"SOME_VAR=ON -DOTHER_VAR=ON"`, which is a bit weird.

Reply:
@christophebedard Addressed in the new commit https://github.com/ros-tooling/ros-github-scripts/pull/11/commits/206a33a64c74a52335ce8198f7580d7889894206
Also updated help section.